### PR TITLE
Problem:(Fix #1) Outdated content

### LINKED ISSUE #1: Problem: Outdated content
Remove outdated content from the chain docs such as the old deployment steps and Rust specific modules.

### PR DESCRIPTION
**Pull request by lezzokafka**
Check list:
- [x] Overview on Crypto.com Chain ;
- [x] Architecture;
- [x] Genesis file explained ;
- [x] Network parameters explained ;
- [x] Chain ID;
- [x] Account prefixes.

Cosmos SDK modules (#11)

- [x] bank
- [x] distribution
- [x] gov
- [x] mint
- [x] slashing
- [x] staking
----
- [x] Nav bar
---
#34 will be resolved in a separated task

### CODE REVIEW

File: docs/chain-details/architecture.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Architecture
+
+## Overview
+
+Building a blockchain is not just about software/hardware development. Instead, it is a combination of technological design, incentive mechanism, game theory and governance, which together nourish a robust system that also allows for continuous innovation. Our initially proposed architecture may hence undergo future revisions in response to changes in incentives, governance or other external requirements.
+
+Crypto.com Chain is open to the public to join, participate and scrutinise related transactions. We do not expect that, for example, mobile clients will be able to perform heavy-lifting tasks and have a reliable always-online network connection. For that reason, there are two different types of nodes are responsible for various duties:
+
+- Validator (Council Node), responsible for validating and committing new blocks to the blockchain; and
+- Community Node (Full node), responsible for fetching the blockchain data and serve it upon the client's request.
```
Comment (tomtau) [inline docs/chain-details/architecture.md:11]: does introducing special terminology help anything? just use "full node" and "validator"

File: docs/chain-details/chain-id.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+[BIP-0173](https://github.com/satoshilabs/slips/blob/master/slip-0173.md) defines a new format for segregated witness output addresses that contains a human-readable part that identifies the coin type. Crypto.com chain has different address prefixes for its corresponding network types, these prefixes are:
+
+| Mainnet | Testnet | Devnet |
+| ------- | ------- | ------ |
+| `cro`   | `tcro`  | `dcro` |
+
+Crypto.com chain uses the Bech32 address format wherever users must handle binary data. Bech32 encoding provides robust integrity checks on data and the human readable part(HRP) that provides contextual hints that can assist UI developers with providing informative error messages. Specifically, we have the following HRP prefix for different addresses types in the mainnet:
+
+|                    | Address bech32 Prefix | Pubkey bech32 Prefix |
```
Comment (tomtau) [inline docs/chain-details/chain-id.md:19]: pubkey bech32 representation may be removed in the SDK release

File: docs/chain-details/introduction.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Crypto.com Chain 
+
+We believe that enabling cryptocurrency spending in the real world will constitute an adoption catalyst. Current traditional payment institutions and existing blockchains have failed to provide a secure, scalable and decentralized solution to support cryptocurrency payment. The key limitations of existing blockchain-powered payment networks are that they are:
+
+- Too complex to setup and use;
+- Not friendly to crypto first timers; and
+- Rarely supported beyond their own blockchain;
+
+Accordingly, we propose Crypto.com Chain, the next generation decentralized mobile payment protocol, the most efficient and secure way to pay and be paid in crypto, anywhere, any crypto at little to no cost. Crypto.com Chain will deliver on its vision by developing innovative technology components and processes catered specifically to cryptocurrency payment, while leveraging proven blockchain technology structural design elements.
```
Comment (tomtau) [inline docs/chain-details/introduction.md:21]: is the re-pivoted version still mobile payment-focused? probably sync with business white paper what it says?

File: docs/chain-details/module_distribution.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+The `distribution` module is responsible for the distribution of rewards to the validators and delegators.
+
+### Overview
+
+#### Network Parameters
+
+Below are all the network parameters for the `distribution` module:
+
+- `community_tax` - The rate of community tax;
```
Comment (tomtau) [inline docs/chain-details/module_distribution.md:13]: I guess in this module description, I guess it'll be good to explain/note why community pool may be non-zero even if the tax rate is zero.

File: docs/chain-details/module_overview.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Modules - overview
+
```
Comment (tomtau) [inline docs/chain-details/module_overview.md:2]: Devashish recently added "module" for the supply querying, so perhaps good to document that too?

File: docs/chain-details/module_slashing.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+  A validator is said to be **non-live** when they fail to sign at least `min_signed_per_window` blocks (in percentage) in the
+  last `signed_blocks_window` blocks successfully. `signed_blocks_window` and `min_signed_per_window` are network
+  parameters and can be configured during genesis and can be updated during runtime by the governance module.
+
+:::tip Example:
+For example, if `block_signing_window` is `2000` blocks and `min_signed_per_window` is `0.5`, a validator will
+be marked as **non-live** and jailed if they fail to successfully sign at least `2000*0.5=1000` blocks in last `2000` blocks.
+:::
+
+- Byzantine Faults (Double signing)
```
Comment (tomtau) [inline docs/chain-details/module_slashing.md:36]: I think in the latest tendermint/sdk, there may be one other fault type defined?

**Review by tomtau — CHANGES_REQUESTED** [review]
looks all right -- the intro sections may be shortened / synced up with whatever is in the business white paper

File: docs/chain-details/chain-id.md (hunk 2 above)
Comment (lezzokafka) [reply docs/chain-details/chain-id.md:19]: Yes, I will remove the `Pubkey bech32 Prefix` here and keep an eye on https://github.com/cosmos/cosmos-sdk/issues/7447.

File: docs/chain-details/architecture.md (hunk 1 above)
Comment (lezzokafka) [reply docs/chain-details/architecture.md]: [Updated](https://github.com/crypto-com/chain-docs/pull/38/commits/89c4e43c18b989be0b84687afa780187dfe932ee#diff-14fb79233d19786b43ce2f4bcff6ba5bfdef6c775705f8f0437b2c8a52c0e002R9) to "full node" and "validator"

File: docs/chain-details/introduction.md (hunk 3 above)
Comment (lezzokafka) [reply docs/chain-details/introduction.md]: [Aligned](https://github.com/crypto-com/chain-docs/pull/38/commits/89c4e43c18b989be0b84687afa780187dfe932ee#diff-37b5ab464a89b7d9971b077c76530f03e2f9ef228dc5652967508ac0f745b1c5R9) with the business white paper.

File: docs/chain-details/module_distribution.md (hunk 4 above)
Comment (lezzokafka) [reply docs/chain-details/module_distribution.md:13]: Added further details on [Community tax](https://github.com/crypto-com/chain-docs/pull/38/commits/89c4e43c18b989be0b84687afa780187dfe932ee#diff-908b614c2992c840fd286f4c491cc499dd1cbf5b52bc3a868d1adf6b3ffdbf8fR46).

File: docs/chain-details/module_overview.md (hunk 5 above)
Comment (lezzokafka) [reply docs/chain-details/module_overview.md:2]: Sure, I have included the supply module in https://github.com/crypto-com/chain-docs/pull/38/commits/b9e6333043961f87d336f7245182d9d4c38e407a.

**Review by calvinaco — APPROVED** [review]

**Pull request merged**